queries.yaml

```diff
@@ -251,3 +251,9 @@ post_process:
       items_matching:
         kind: station
       rank_key: tile_rank
+  - fn: TileStache.Goodies.VecTiles.transform.remove_duplicate_features
+    params:
+      source_layers: [pois, buildings, landuse_labels]
```

Comment:
> I forget why we are duplicating this information anyway?

When we started packing label positions into their source layer, we had to also continue supporting the legacy `landuse-labels` layer so developers wouldn't see regressions. But in the release notes we said we'd remove `landuse-labels` after a few releases. 

That sounds more like a breaking v1.0 change to me, so in the mean time we should copy them in a later step?